tests/integration/global-content-repo/GlobalContentRepoTest.php

```diff
@@ -0,0 +1,19 @@
+<?php // phpcs:disable
```

Comment:
I think it's worth following our coding standards even in the tests, I know the generator outputs the stub differently but there's not loads to edit to bring things in line.